openpype/hosts/max/plugins/publish/validate_tyflow_data.py

```diff
@@ -0,0 +1,91 @@
+import pyblish.api
+from openpype.pipeline import PublishValidationError
+from pymxs import runtime as rt
+
+
+class ValidateTyFlowData(pyblish.api.InstancePlugin):
+    """Validate that TyFlow plugins or
+    relevant operators being set correctly."""
+
+    order = pyblish.api.ValidatorOrder
+    families = ["pointcloud", "tycache"]
+    hosts = ["max"]
+    label = "TyFlow Data"
+
+    def process(self, instance):
+        """
+        Notes:
+            1. Validate the container only include tyFlow objects
+            2. Validate if tyFlow operator Export Particle exists
+
+        """
+
+        invalid_object = self.get_tyflow_object(instance)
+        if invalid_object:
+            self.log.error(f"Non tyFlow object found: {invalid_object}")
+
+        invalid_operator = self.get_tyflow_operator(instance)
+        if invalid_operator:
+            self.log.error(
+                "Operator 'Export Particles' not found in tyFlow editor.")
+        if invalid_object or invalid_operator:
+            raise PublishValidationError(
+                "issues occurred",
+                description="Container should only include tyFlow object\n "
+                "and tyflow operator 'Export Particle' should be in \n"
+                "the tyFlow editor")
```

Comment:
The new lines `\n` seem to be in odd locations here mid sentences.

```suggestion
                description="Container should only include tyFlow object "
                "and tyflow operator 'Export Particle' should be in "
                "the tyFlow editor.")
```